core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -319,15 +319,17 @@ public void onSwipeBottom() {
       public void onSwipeLeft() {
         if (currentWebViewIndex < webViewList.size() - 1) {
           View current = getCurrentWebView();
-          current.setAnimation(transitionLeft());
+          current.startAnimation(AnimationUtils.loadAnimation(getApplicationContext(), R.anim.transition_left));
```

Comment:
good but now we have 4 call sites where we load an Animation and it isn't very DRY, try your hand at automatically refactoring this method, `ctrl+m` with your cursor on this method.

Also don't use the application context unless you need to, it is generally best practice to use the closest context possible, that is mostly when it comes to theming but there are other differences among the contexts. Here the view has a context, retrievable by `View.getContext` or you can simply use `this` as an Activity is also a context

Reply:
@macgills is it fine now?